spec/models/user_spec.rb

```diff
@@ -162,6 +162,8 @@
     let(:non_affiliate) { users(:developer) }
     let(:not_active_user) { users(:not_active_user) }
     let(:active_user) { users(:active_user) }
+    let(:never_active_user) { users(:never_active_user)}
+    let(:new_non_active_user) { users(:new_non_active_user)}
```

Comment:
Needs a space before the closing curly bracket: `{ users(:new_non_active_user) }`